website/templates/project/settings.mako

```diff
@@ -132,15 +130,14 @@
                         <form id="selectAddonsForm">
 
                             % for category in addon_categories:
-
                                 <%
-                                    addons = [
-                                        addon
-                                        for addon in addons_available
-                                        if category in addon.categories
-                                    ]
+                                    addons = []
```

Comment:
Reusing Python code, i.e. using view helper functions, will be *much* simpler than trying to reuse mako code.